### PR TITLE
storage: remove test file after test

### PR DESCRIPTION
**Pull request by gyuho**
/cc @xiang90

### CODE REVIEW

File: storage/kv_test.go [hunk 1]
```diff
@@ -707,10 +707,12 @@ func TestKVSnapshot(t *testing.T) {
 		{Key: []byte("foo2"), Value: []byte("bar2"), CreateRevision: 3, ModRevision: 3, Version: 1, Lease: 3},
 	}
 
-	f, err := os.Create("new_test")
+	newPath := "new_test"
+	f, err := os.Create(newPath)
 	if err != nil {
 		t.Fatal(err)
 	}
+	defer os.Remove(newPath)
```
Comment (xiang90) [inline storage/kv_test.go]: blank line between 715 and 716?

**Comment by xiang90** [comment]
LGTM

File: storage/kv_test.go (hunk 1 above)
Comment (gyuho) [inline storage/kv_test.go]: Ok

**Comment by gyuho** [comment]
Just added. Will merge after CI passes.

**Pull request merged**